digital_asset_types/src/dapi/common/asset.rs

```diff
@@ -479,6 +479,10 @@ pub fn asset_to_rpc(asset: FullAsset, options: &Options) -> Result<RpcAsset, DbE
                 .creator_hash
                 .map(|e| if asset.compressed { e.trim() } else { "" }.to_string())
                 .unwrap_or_default(),
+            asset_data_hash: asset
+                .asset_data_hash
+                .map(|e| if asset.compressed { e.trim() } else { "" }.to_string()),
```

Comment:
Do we really need to use empty String in case of None, since we use skip_serializing_if in the model.

Reply:
I'm just doing it to be consistent with the other hsahes.